modules/ROOT/pages/operating-about.adoc

```diff
@@ -11,10 +11,13 @@ Mule app developers in your organization who create Mule apps through the xref:s
 
 The following topics describe how to manage Anypoint Platform PCE after installation and configuration is complete:
 
+* Close port 61009 in the cluster security group. If you are using AWS, you can use the AWS Web console to do this.
```

Comment:
GLOBAL bullet-list item style, max. one sentence:
I looked this up as well, and could find only "AWS Management Console"
```suggestion
* Close port 61009 in the cluster security group. +
If you are using AWS, you can use the AWS Management Console to do this.
```
OR
```suggestion
* Close port 61009 in the cluster security group. +
If you are using AWS, you can use the AWS web console to do this.
```